index.html

```diff
@@ -428,7 +428,7 @@ <h2 class="white-header">FOSSASIA on social media</h2>
             <!-- Stop ignoring -->
             <!-- Start ignoring BootLintBear -->
                 <div class="col-md-3">
-                    <a class="twitter-timeline" data-width="370"
+                    <a class="twitter-timeline" data-width="370" data-height = "500"
```

Comment:
That's before:
![6](https://cloud.githubusercontent.com/assets/6567844/21293265/dd87eabc-c527-11e6-90db-4078d57fb95b.png)

That's after:
![image](https://cloud.githubusercontent.com/assets/6567844/21293270/fd55cd96-c527-11e6-955c-91622d2484f3.png)

@animeshsinghweb